src/js/views/components/progressBar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import _ from 'lodash';
+
+
+let ProgressBar = React.createClass({
+	render() {
```

Comment:
Good point! I'm also going to add a check to make sure noone supplies a progress over 100%... because that might cause some issues!